NTO/Knowledge/attributes/archived.ttl

```diff
@@ -0,0 +1,15 @@
+@prefix ogit:                   <http://www.purl.org/ogit/> .
+@prefix owl:                    <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs:                   <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix dcterms:                <http://purl.org/dc/terms/> .
+
+ogit:archived
```

Comment:
this would define ogit/archived, not ogit/Knowledge/archived, please add proper prefix for ogit/Knowledge NTO here